partials/tabs/home.html

```diff
@@ -115,7 +115,7 @@ <h2 class="fine">get started</div>
                     <div btf-markdown ng-include="'data/snippets/' + sc.snip[sc.lang_selected]" class="card card-content"></div>
                     <p>
                         Need more powerful analysis?
-                        <a ng-click="tc.setTab('/languages')">
+                        <a onclick="$('#tab-languages').trigger('click')">
```

Comment:
oh you make that url redirect to /#!languages?
ok, I understand now, so how about the ``id``?
``id`` does not make sense if I change it.